dex/testing/dcrdex/harness.sh

```diff
@@ -26,6 +26,16 @@ go build -o ${DCRDEX_DATA_DIR}/dcrdex -ldflags \
 EOF
 chmod +x "${DCRDEX_DATA_DIR}/build"
 
+cat > "${DCRDEX_DATA_DIR}/build-lock" <<EOF
```

Comment:
This script is so we can easily rebuild with usable lock times with just `./build-lock` or arbitrary taker/maker lock times like `./build-lock 2m 4m`